packages/terra-status-view/docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Terra Status View
+
+The status view component provides a customizable icon, message, and buttons in a view.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-status-view`
+  - `yarn add terra-status-view`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Button from 'terra-button';
+import StatusView from 'terra-status-view';
+
+<StatusView
+  isAlignedTop={false}
+  isGlyphHidden={false}
+  variant={StatusView.Opts.variants.ERROR}
+  title="A 500 error was thrown from the service."
+  message=message="A null pointer exception was thrown."
+>
+  <Button text="OK" key="1 />
```

Comment:
Given buttons are the expected children, we should update this to be

`< StatusView.Button text="OK" key="1 />`

Reply:
I'm cool with leaving this as is using `<Button />` instead of `<StatusView.Button />`